spec/features/authentication_spec.rb

```diff
@@ -14,6 +16,11 @@
           )
         )
       end
+
+      scenario 'changes the login text' do
+        expect(page).not_to have_content('Sign In With Slack')
```

Comment:
I think we should be using i18n here. That's my bad - I should have done that when I put in that navbar item.